### DISCUSSION TITLE
The current project is by default editable?

### DISCUSSION THREAD

**Discussion by halloleo** [python-poetry/poetry#4793] [Q&A] [answered]
When I set my project up with Poetry and I do `poetry install` my project script is installed and I can run it with:

```
poetry run myscript
```

So far so good. When I now change the source code to the file were the entry point `myscript` is defined (myscript.py) these changes get picked up immediately without another  `poetry install` . So I assume my project is not really installed _into_ the project's virtual environment but somehow referenced. So it is _editable_.

However, when I use pipx to install the project separately, the code for my script gets frozen/baked into the virtual environment.

Both is exactly what I want. I just like to ask whether I can always rely on the editable behaviour in my project.

**Comment by clintonroy** [comment, marked as answer]
At least on the poetry side: https://python-poetry.org/docs/basic-usage/ says
``The current project is installed in editable mode by default.''

**Reply by halloleo** [reply]
Cool. Thanks. Makes total sense.
